src/Modem.h

```diff
@@ -83,6 +86,8 @@ class ModemClass {
   #define MAX_URC_HANDLERS 10 // 7 sockets + GPRS + GSMLocation + GSMVoiceCall
   static bool _debug;
   static ModemUrcHandler* _urcHandlers[MAX_URC_HANDLERS];
+
+  Stream* _debugStream;
```

Comment:
I think this can be of type [Print](https://github.com/arduino/ArduinoCore-samd/blob/master/cores/arduino/Print.h) instead of Stream.

Reply:
Good catch, should I rename method to setDebugPrint?

Reply:
Also I will probably add `static` keyword since `_debug` flag is also static